src/replayToMovie.js

```diff
@@ -172,6 +164,17 @@ module.exports.renderImages = async (replay, writer, parentSegment) => {
   let lastBackground;
   let lastForeground;
 
+  // Create our emulated canvas.
+  // We have to create a new canvas on every request, otherwise under periods
+  // of high traffic the canvas can get into a state where it "freezes" and
+  // repeats a single frame for the length of an entire video.
```

Comment:
Code change looks good to me - can you add a link to the issue? https://github.com/code-dot-org/dance-party/issues/514 (I added a summary of our investigation there.)